README.md

```diff
@@ -48,7 +48,7 @@ most recent version executing the `symfony self-update` command.
 Usage
 -----
 
-If you have PHP 5.4 or higher, there is no need to configure a virtual host
+If you have PHP `5.4` or higher, there is no need to configure a virtual host
```

Comment:
I would simply omit the first part of the sentence and start it with "There is […]".